### PR TITLE
Add mpeth token

### PR DESCRIPTION
**Pull request by Calcifer1001**

### CODE REVIEW

File: tokens/mpeth.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0x48afbbd342f64ef8a9ab1c143719b63c2ad81710",
+    "near_address": "48afbbd342f64ef8a9ab1c143719b63c2ad81710.factory.bridge.near",
+    "aurora_address": "0xb64079dc31e0b28919e506d00f5cc50a74e7cabb",
+    "name": "MetaPoolETH",
+    "symbol": "MPETH",
+    "decimals": 18,
+    "icon": "data:image/svg+xml,%3Csvg%20width%3D%22204%22%20height%3D%22204%22%20viewBox%3D%220%200%20204%20204%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Crect%20x%3D%220.25%22%20y%3D%220.235352%22%20width%3D%22203.501%22%20height%3D%22203.501%22%20rx%3D%22101.751%22%20fill%3D%22%23BAC2E2%22%2F%3E%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M102.001%2039.6226L86.8763%2054.54L102.001%2069.4575L117.126%2054.54L102.001%2039.6226ZM77.8967%2063.2419L63.257%2077.9581L101.794%20116.696L140.331%2077.9581L125.691%2063.242L101.794%2087.2638L77.8967%2063.2419ZM54.6541%2086.8613L39.6377%20101.899L102.001%20164.349L164.364%20101.899L149.348%2086.8614L102.001%20134.274L54.6541%2086.8613Z%22%20fill%3D%22%230C2246%22%2F%3E%3C%2Fsvg%3E",
```
Comment (karim-en) [inline tokens/mpeth.json:8]: Could you please use base64 encoding.
https://codebeautify.org/svg-to-base64-converter

**Review by karim-en — CHANGES_REQUESTED** [review]

File: tokens/mpeth.json (hunk 1 above)
Comment (Calcifer1001) [reply tokens/mpeth.json]: Sure. Let me know if it is fine now. Thanks!

**Comment by Calcifer1001** [comment]
Hey @karim-en. Can I do something to move this forward faster? Thank you!

File: tokens/mpeth.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0x48afbbd342f64ef8a9ab1c143719b63c2ad81710",
+    "near_address": "48afbbd342f64ef8a9ab1c143719b63c2ad81710.factory.bridge.near",
+    "aurora_address": "0xb64079dc31e0b28919e506d00f5cc50a74e7cabb",
+    "name": "MetaPoolETH",
+    "symbol": "MPETH",
+    "decimals": 18,
+    "icon": "PHN2ZyB3aWR0aD0iMjA0IiBoZWlnaHQ9IjIwNCIgdmlld0JveD0iMCAwIDIwNCAyMDQiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+DQo8cmVjdCB4PSIwLjI1IiB5PSIwLjIzNTM1MiIgd2lkdGg9IjIwMy41MDEiIGhlaWdodD0iMjAzLjUwMSIgcng9IjEwMS43NTEiIGZpbGw9IiNCQUMyRTIiLz4NCjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMTAyLjAwMSAzOS42MjI2TDg2Ljg3NjMgNTQuNTRMMTAyLjAwMSA2OS40NTc1TDExNy4xMjYgNTQuNTRMMTAyLjAwMSAzOS42MjI2Wk03Ny44OTY3IDYzLjI0MTlMNjMuMjU3IDc3Ljk1ODFMMTAxLjc5NCAxMTYuNjk2TDE0MC4zMzEgNzcuOTU4MUwxMjUuNjkxIDYzLjI0MkwxMDEuNzk0IDg3LjI2MzhMNzcuODk2NyA2My4yNDE5Wk01NC42NTQxIDg2Ljg2MTNMMzkuNjM3NyAxMDEuODk5TDEwMi4wMDEgMTY0LjM0OUwxNjQuMzY0IDEwMS44OTlMMTQ5LjM0OCA4Ni44NjE0TDEwMi4wMDEgMTM0LjI3NEw1NC42NTQxIDg2Ljg2MTNaIiBmaWxsPSIjMEMyMjQ2Ii8+DQo8L3N2Zz4NCg==",
```
Comment (olga24912) [inline tokens/mpeth.json:8]:
```suggestion
    "icon": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjA0IiBoZWlnaHQ9IjIwNCIgdmlld0JveD0iMCAwIDIwNCAyMDQiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxyZWN0IHg9IjAuMjUiIHk9IjAuMjM1MzUyIiB3aWR0aD0iMjAzLjUwMSIgaGVpZ2h0PSIyMDMuNTAxIiByeD0iMTAxLjc1MSIgZmlsbD0iI0JBQzJFMiIvPgo8cGF0aCBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTEwMi4wMDEgMzkuNjIyNkw4Ni44NzYzIDU0LjU0TDEwMi4wMDEgNjkuNDU3NUwxMTcuMTI2IDU0LjU0TDEwMi4wMDEgMzkuNjIyNlpNNzcuODk2NyA2My4yNDE5TDYzLjI1NyA3Ny45NTgxTDEwMS43OTQgMTE2LjY5NkwxNDAuMzMxIDc3Ljk1ODFMMTI1LjY5MSA2My4yNDJMMTAxLjc5NCA4Ny4yNjM4TDc3Ljg5NjcgNjMuMjQxOVpNNTQuNjU0MSA4Ni44NjEzTDM5LjYzNzcgMTAxLjg5OUwxMDIuMDAxIDE2NC4zNDlMMTY0LjM2NCAxMDEuODk5TDE0OS4zNDggODYuODYxNEwxMDIuMDAxIDEzNC4yNzRMNTQuNjU0MSA4Ni44NjEzWiIgZmlsbD0iIzBDMjI0NiIvPgo8L3N2Zz4K",
```

Comment (olga24912) [reply tokens/mpeth.json:8]: I got this result  after the base64 conversion. Please check.

**Review by olga24912 — CHANGES_REQUESTED** [review]

**Comment by Calcifer1001** [comment]
@olga24912 thank you for your review. I've just commited it!

**Review by karim-en — APPROVED** [review]

**Pull request merged**